src/App.js

```diff
@@ -1,12 +1,31 @@
 import BookList from './components/book/BookList';
 import './App.css';
+import { Link, Route, Switch } from 'react-router-dom';
+import BookDetail from './components/book/BookDetail';
 
 function App() {
   // TODO: Add routes to books & views
   return (
     <main className="container">
       <h1>Library Catalog</h1>
-      <BookList />
+      <h2>I see that you have made it to my catalog</h2>
```

Comment:
lines 11 and 12 should be in a separate component, inside of a route, because when i click the link i don't want to keep seeing them — i want that to switch out for the list of books